src/listener/server-hardware.js

```diff
@@ -179,14 +179,15 @@ export default class ServerHardwareListener extends Listener {
     let status = msg.status.toLowerCase();
     status = status.charAt(0).toUpperCase() + status.slice(1);
     this.client.ServerHardware.getHardwareByStatus(status).then((res) => {
-      if (res.count === 0) {
-        return this.transform.text(msg, msg.message.user.name + ", I didn't find any blades with a " + msg.status.toLowerCase() + " status.");
+
+      if (res.members.length === 0) {
+        return this.transform.text(msg, msg.message.user.name + ", I didn't find any blades with a " + status.toLowerCase() + " status.");
       }
       else {
-        if (msg.status.toLowerCase() === "ok") {
-          return this.transform.send(msg, res, "Okay " + msg.message.user.name + ", the following blades have an " + msg.status.toUpperCase() + " status.");
+        if (status.toLowerCase() === "ok") {
+          return this.transform.send(msg, res, "Okay " + msg.message.user.name + ", the following blades have an " + status.toLowerCase() + " status.");
```

Comment:
Why lower case on the status in the message to the user?

Reply:
Fixed!